clang/tools/include-mapping/cppreference_parser.py

```diff
@@ -69,7 +69,9 @@ def _ParseSymbolPage(symbol_page_html, symbol_name):
                 was_decl = True
                 # Symbols are in the first cell.
                 found_symbols = row.find("td").stripped_strings
-                if not symbol_name in found_symbols:
+                if not any(
+                    sym == symbol_name or sym == qual_name for sym in found_symbols
+                ):
```

Comment:
Cannot use `if not (symbol_name in found_symbols or qual_name in found_symbols)`, because `found_symbols)` is a generator, not a list.